.travis.yml

```diff
@@ -49,18 +52,25 @@ install:
   - pip install -r requirements.txt
   - pip install .
   - pip install coveralls
+  # install vaxrank and its dependent data
+  - pip install --upgrade -r $PWD/vaxrank/requirements.txt
+  - pyensembl install --release 87 --species human
+  - pyensembl install --release 87 --species mouse
 before_script:
   # needed for MHCflurry predictor test
   - mhcflurry-downloads fetch
 script:
   - ./lint.sh
-  - nosetests -sv test --with-coverage --cover-package=mhctools
+  # -a '!skip' lets us skip certain tests on travis (relevant for vaxrank)
+  # this command runs both mhctools and vaxrank tests
+  - nosetests -sv test --with-coverage --cover-package=mhctools -a '!skip' $PWD/vaxrank
 after_success:
   coveralls
 deploy:
   provider: pypi
+  distributions: sdist
```

Comment:
modeling this one on topiary - that travis config is set up to auto-deploy to PyPi on an __init__.py version bump, and this is the only difference